app/controllers/development_metrics_controller.rb

```diff
@@ -165,10 +165,6 @@ def action
     @action ||= params[:action]
   end
 
-  def code_climate_repository_summary
-    CodeClimateSummaryRetriever.call(repository.id)
```

Comment:
is this class still in use or does it need to be removed?

Reply:
No, I can remove it 💪